src/azure-cli/azure/cli/command_modules/extension/__init__.py

```diff
@@ -64,6 +64,8 @@ def load_arguments(self, command):
                        arg_group='Experimental Pip')
             c.argument('pip_extra_index_urls', options_list=['--pip-extra-index-urls'], nargs='+',
                        help='Space-separated list of extra URLs of package indexes to use. This should point to a repository compliant with PEP 503 (the simple repository API) or a local directory laid out in the same format.', arg_group='Experimental Pip')
+            c.argument('install_setup_extras', options_list=['--install-extras'], nargs='+',
```

Comment:
The word "install" is basically a synonym of "setup". The naming of `install_setup_extras` seems inappropriate. Why does it have a different name than `--install-extras`?

Reply:
Fixed